api/node/package.json

```diff
@@ -1,25 +1,49 @@
 {
-    "name": "slint-ui",
-    "version": "1.3.0",
-    "homepage": "https://github.com/slint-ui/slint",
-    "license": "SEE LICENSE IN LICENSE.md",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/slint-ui/slint"
-    },
-    "main": "dist/index.js",
-    "types": "dist/index.d.ts",
-    "dependencies": {
-        "@types/node": "^14.11.11",
-        "neon-cli": "^0.4",
-        "typescript": "^4.0.3"
-    },
-    "scripts": {
-        "install": "neon build --release && tsc",
-        "build": "tsc",
-        "docs": "typedoc --hideGenerator --readme cover.md lib/index.ts"
-    },
-    "devDependencies": {
-        "typedoc": "^0.19.2"
+  "name": "slint-ui",
+  "version": "1.3.0",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "license": "GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial",
+  "napi": {
+    "name": "slint-ui"
+  },
```

Comment:
Do you know which part of napi-rs exactly needs this field? I wonder if this perhaps only needed for binary packages.

Reply:
It looks like, that it works without it. @tronical should I remove it?

Reply:
Yes. I'm in favor of keeping package.json minimal and only have fields that we need.

Reply:
Me ether ;-). Is removed.